hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -279,25 +278,25 @@ spec = around (showLogsOnFailure "EndToEndSpec") $ do
 
     describe "restarting nodes" $ do
       it "can abort head after restart" $ \tracer -> do
-        withClusterTempDir "abort-after-restart" $ \tmpDir -> do
+        withClusterTempDir $ \tmpDir -> do
           withCardanoNodeDevnet (contramap FromCardanoNode tracer) tmpDir $ \node ->
             publishHydraScriptsAs node Faucet
               >>= restartedNodeCanAbort tracer tmpDir node
 
       it "can observe a commit tx after a restart, even when a tx happened while down" $ \tracer -> do
-        withClusterTempDir "commit-after-restart" $ \tmpDir -> do
+        withClusterTempDir mmit-after-restart" $ \tmpDir -> do
           withCardanoNodeDevnet (contramap FromCardanoNode tracer) tmpDir $ \node ->
             publishHydraScriptsAs node Faucet
               >>= restartedNodeCanObserveCommitTx tracer tmpDir node
 
       it "prevent resuming a head after reconfiguring a peer" $ \tracer -> do
-        withClusterTempDir "prevent-resume-reconfiguring-peer" $ \tmpDir -> do
+        withClusterTempDir event-resume-reconfiguring-peer" $ \tmpDir -> do
```

Comment:
I think you probably did a mass-rename here or something that caused the starting quote to dissapear?

Reply:
The string should not be passed in the fst place

Reply:
This also appears to happen in many places

Reply:
Fix this now.